openwisp_monitoring/device/tests/test_models.py

```diff
@@ -590,6 +590,16 @@ def test_ok_problem_problem_problem_ok(self):
         load.check_threshold(80)
         self.assertEqual(dm.status, 'ok')
 
+    def test_management_ip_clear_device_offline(self):
```

Comment:
Should there be a test case too to ensure management_ip is not cleared when `AUTO_CLEAR_MANAGEMENT_IP` is false?

Reply:
Yes, that would be great to have.